### PR TITLE
refactor: set consistent folder structure inside audittrail sorage

### PR DESCRIPTION
**Pull request by adamdecaf**
The same change as https://github.com/moov-io/achgateway/pull/122 but fixed for CI.

### CODE REVIEW

**Comment by zbruhnke** [comment]
only thing I would point out is that does not actually standardize the structure because you do not have a directory on the file originated in storage so it will just be date then filename, I find this to be fine for at least our use cases, but just wanted to call out that there is a difference between files received and files originated and how they're stored here still

File: internal/incoming/odfi/processor.go [hunk 1]
```diff
@@ -150,9 +152,12 @@ func processFile(path string, auditSaver *AuditSaver, fileProcessors Processors)
 	dir, filename := filepath.Split(path)
 	dir = filepath.Base(dir)
 
+	// Allow an override of root path or use legacy path
+	folder := strx.Or(os.Getenv("RDFI_FOLDER"), "odfi")
```
Comment (adamdecaf) [inline internal/incoming/odfi/processor.go:156]: This name confuses me. We're storing files from the ODFI and potentially as the RDFI here. I'd rather move forward with the ODFI/RDFI split than merge this into v0.16.0

File: internal/incoming/odfi/processor.go [hunk 2]
```diff
@@ -150,9 +152,12 @@ func processFile(path string, auditSaver *AuditSaver, fileProcessors Processors)
...
+
 	// Persist the file if needed
 	if auditSaver != nil {
-		path := fmt.Sprintf("odfi/%s/%s/%s/%s", auditSaver.hostname, dir, time.Now().Format("2006-01-02"), filename)
+		path := fmt.Sprintf("%s/%s/%s/%s/%s", folder, auditSaver.hostname, time.Now().Format("2006-01-02"), dir, filename)
```
Comment (adamdecaf) [inline internal/incoming/odfi/processor.go:160]: This breaks ach-web-viewer as it is today, so I'm hesitant to merge this change. I know it's more convenient but we have precedent already.

**Comment by adamdecaf** [comment]
I'm going to work on the ODFI/RDFI change in https://github.com/moov-io/achgateway/pull/123 instead of merging this breaking change into v0.16.0.

**Pull request closed without merging**